ee/src/main/java/org/jboss/as/ee/subsystem/ManagedExecutorServiceResourceDefinition.java

```diff
@@ -178,19 +190,30 @@
             .addMetric(ManagedExecutorServiceMetricsAttributes.THREAD_COUNT_AD, (context, service) -> context.getResult().set(service.getExecutorService().getRuntimeStats().getThreadsCount()))
             .build();
 
+    /**
+     * terminate hung threads op
+     */
+    private static final ManagedExecutorTerminateHungTasksOperation<ManagedExecutorServiceService> TERMINATE_HUNG_TASKS_OP = new ManagedExecutorTerminateHungTasksOperation<>(CAPABILITY, RESOLVER, service -> service.getExecutorService());
+
     /**
      *
      */
     private final boolean registerRuntimeOnly;
 
     ManagedExecutorServiceResourceDefinition(boolean registerRuntimeOnly) {
-        super(new SimpleResourceDefinition.Parameters(PATH_ELEMENT, EeExtension.getResourceDescriptionResolver(EESubsystemModel.MANAGED_EXECUTOR_SERVICE))
+        super(new SimpleResourceDefinition.Parameters(PATH_ELEMENT, RESOLVER)
                 .setAddHandler(ManagedExecutorServiceAdd.INSTANCE)
                 .setRemoveHandler(new ServiceRemoveStepHandler(ManagedExecutorServiceAdd.INSTANCE))
                 .addCapabilities(CAPABILITY));
         this.registerRuntimeOnly = registerRuntimeOnly;
     }
 
+    @Override
+    public void registerOperations(ManagementResourceRegistration resourceRegistration) {
+        super.registerOperations(resourceRegistration);
+        TERMINATE_HUNG_TASKS_OP.registerOperation(resourceRegistration);
```

Comment:
This is contrary to https://github.com/wildfly/wildfly-proposals/pull/272/files#diff-92d9b62a418fa48fe8ae1baf9370be6f000f7c8901f0c459b836ff48444941f5R90

Guarding this with an 'if (resourceRegistration.getProcessType().isServer())' check will correct that.

Reply:
Fixed, did it inside TERMINATE_HUNG_TASKS_OP.registerOperation(resourceRegistration) tho, since that applies to both executor resource definitions.